### PR TITLE
Install extension no longer supported via azure portal

### PR DESCRIPTION
**Pull request by NAshwini**
Signed-off-by: NAshwini <ashwini.nehate@msystechnologies.com>

### Description

Installing the extension is no longer supported through the Azure Portal. Hence removed the documentation and replace it with a warning that it is no longer supported.

https://docs.chef.io/azure_portal/

### Issues Resolved

https://github.com/chef-partners/azure-chef-extension/issues/292#

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by NAshwini** [comment]
@btm can you please review it. Thank you.

**Comment by IanMadd** [comment]
It would be best if don't direct users to docs pages on github. Can we link to relevant pages on docs.chef.io or update content if it's not already there?

**Comment by NAshwini** [comment]
> It would be best if don't direct users to docs pages on github. Can we link to relevant pages on docs.chef.io or update content if it's not already there?
 
Thank you @IanMadd. Will work on it.

**Comment by IanMadd** [comment]
Hi @NAshwini can you [give me permission](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) to edit this PR. There are a few edits that I want to make. Thanks!

**Comment by NAshwini** [comment]
> Hi @NAshwini can you [give me permission](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) to edit this PR. There are a few edits that I want to make. Thanks!

Hi @IanMadd I have gone through PR but for me I am not able to get such type of checkbox "Allow edits from maintainers." where I can give you the permission. It seems that I don't have enough permission on this repo.

File: content/azure_chef_cli.md
Comment (IanMadd) [inline content/azure_chef_cli.md]:
```suggestion
`bootstrap_channel`

:   Specify the [channel](/packages/) for installing the Chef Infra Client version.
    Options are `stable`, `current` or `unstable` release channels.
```

File: content/azure_chef_cli.md
Comment (IanMadd) [inline content/azure_chef_cli.md]:
```suggestion
`chef_package_path`

:   Specifies a local path to install Chef Infra Client from.
    This feature is mainly used for cases where there are restrictions on
    internet access.

    {{< note spaces=4 >}}

    Azure extensions have network access limitations. See the
    [Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/features-linux#network-access)
    for more information.

    {{< /note >}}
```

File: content/azure_chef_cli.md
Comment (IanMadd) [inline content/azure_chef_cli.md]:
```suggestion
`CHEF_LICENSE`

:   Chef Infra Client 15+ requires accepting the CHEF EULA license. Set `CHEF_LICENSE`
    to one of these values `accept`, `accept-silent` or `accept-no-persist`. Refer
    to [CHEF EULA license](/chef_license_accept/#accept-the-chef-eula)
```

File: content/azure_chef_cli.md
Comment (IanMadd) [inline content/azure_chef_cli.md]:
```suggestion
`hints`

:   Specifies the [Ohai Hints](/ohai/#hints) to be set in the Ohai configuration of the target node.
```

File: content/azure_portal.md
Comment (IanMadd) [inline content/azure_portal.md]:
```suggestion
The Chef Extension can no longer be installed through the Azure Portal.
It can be installed using other command line tools, such as:

1. [Azure Powershell cmdlets](/azure_powershell/)
1. [Azure CLI](/azure_chef_cli/)
1. [Knife Azure Plugin](/workstation/knife_azure/)
```

**Review by IanMadd — COMMENTED** [review]
@NAshwini There are a couple other changes I want to make but I can do that in a different PR. Take a look at these suggestions and let me know what you think.

**Comment by NAshwini** [comment]
> @NAshwini There are a couple other changes I want to make but I can do that in a different PR. Take a look at these suggestions and let me know what you think.

Thank you @IanMadd. Made changes as per your suggestion. Please review.

**Comment by IanMadd** [comment]
@NAshwini Everything looks good. Could you rebase the PR and I'll merge it when I get in tomorrow morning. Thanks!

**Comment by Vasu1105** [comment]
@IanMadd @NAshwini is on leave. So I did the rebase. Thank you.

**Review by IanMadd — APPROVED** [review]
Thanks @NAshwini and @Vasu1105

**Pull request merged**
